python-sdk/example_dags/example_dynamic_task_template.py

```diff
@@ -39,4 +44,29 @@
     ).expand(input_file=get_file_list(path=GCS_BUCKET, conn_id=ASTRO_GCP_CONN_ID))
 
     # [END howto_operator_get_file_list]
+
+    # [START howto_operator_get_value_list]
+    @task
+    def custom_task(rating_val):
+        try:
+            print("rating_val ", rating_val)
+            print("rating_val.__dict__ ", rating_val.__dict__)
+            print("rating_val.__dict__.get()", rating_val.__dict__.get("rating"))
+            return float(rating_val.__dict__.get("rating"))
+        except ValueError:
+            pass
```

Comment:
Could we simplify this block to become a single line? It feels it is more complex than needed and may distract from the focus of this example (illustrating how to use `get_file_list`.

Reply:
We want to do some operations with on the result of expanded task so we need this.